.github/actions/common-rc-validation/action.yaml

```diff
@@ -36,19 +36,19 @@ runs:
       shell: bash
       run: |
           echo "---------------------Downloading Python Staging RC----------------------------"
-          wget ${PYTHON_RC_DOWNLOAD_URL}/${RELEASE_VER}/python/apache-beam-${RELEASE_VER}.tar.gz
-          wget ${PYTHON_RC_DOWNLOAD_URL}/${RELEASE_VER}/python/apache-beam-${RELEASE_VER}.tar.gz.sha512
-          if [[ ! -f apache-beam-$RELEASE_VER.tar.gz ]]; then
+          wget ${PYTHON_RC_DOWNLOAD_URL}/${RELEASE_VER}/python/apache_beam-${RELEASE_VER}.tar.gz
+          wget ${PYTHON_RC_DOWNLOAD_URL}/${RELEASE_VER}/python/apache_beam-${RELEASE_VER}.tar.gz.sha512
+          if [[ ! -f apache_beam-$RELEASE_VER.tar.gz ]]; then
```

Comment:
Looks like maybe this is still not working? https://github.com/apache/beam/actions/workflows/run_rc_validation.yml

I like the idea of switching this step to the automated workflow, but I'm guessing it will need more work. I'm fine taking this as a step in the right direction though.

Reply:
Hm, I also thinkthis change won't impact your runs since we're checking out the rc tag/using that version of the composite action

Reply:
I'd suggest just getting rid of this composite action and making it a direct part of the workflow. We're not really saving ourselves much duplication anyways

Reply:
let me merge this and test it again.